src/Guzzle/MiddlewareFactory.php

```diff
@@ -53,7 +53,22 @@ public function retry( $delay = true ) {
 	 * @return callable
 	 */
 	private function getRetryDelay() {
-		return function( $numberOfRetries ) {
+		return function( $numberOfRetries, Response $response = null ) {
+			// The $response argument is only passed as of Guzzle 6.2.2.
+			if( $response !== null ) {
+				// Retry-After may be a number of seonds or an absolute date.
```

Comment:
seonds -> seconds
It also might be nice to include a link to https://www.mediawiki.org/wiki/Manual:Maxlag_parameter here in a comment.